### PR TITLE
Completed Google Calendar Integration

### PR DESCRIPTION
**Pull request by key-joshua**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 src/database/migrations/
 src/database/seeders/
 node_modules/
+
+packageScript.json
```
Comment (Yopo-Nicolas) [inline .gitignore:5]: What is this intended for?

File: src/modules/google/controller/googleController.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    const scopes = JSON.parse(process.env.GOOGLE_SCOPES);
+    const googleAuth = googleOauth2Client.generateAuthUrl({ access_type: 'offline', scope: scopes });
+
+    return res.redirect(googleAuth);
+    } catch (error) {
+        responseUtils.handleError(error.status || httpStatus.INTERNAL_SERVER_ERROR, error.toString());
+        return responseUtils.response(res);
+    }
+};
+
```
Comment (Yopo-Nicolas) [inline src/modules/google/controller/googleController.ts:18]: We should also add google sign in and signup with the firebase authentication

**Review by Yopo-Nicolas — COMMENTED** [review]
Seems okey, just change those and add requested changes

File: .gitignore (hunk 1 above)
Comment (key-joshua) [reply .gitignore:5]: This is for personal use. I have stored my script in `packageScript.json`.

File: src/modules/google/controller/googleController.ts (hunk 2 above)
Comment (key-joshua) [reply src/modules/google/controller/googleController.ts:18]: I will add these two features to JIRA and work on them once I finish my current tasks.

Comment (Yopo-Nicolas) [reply src/modules/google/controller/googleController.ts:18]: if this script is used for migrations, should be in package.json on the scripts part. If it's unrelated to the app it shouldn't be on folder

File: .gitignore (hunk 1 above)
Comment (Yopo-Nicolas) [reply .gitignore:5]: if this script is used for migrations, should be in package.json on the scripts part. If it's unrelated to the app it shouldn't be on folder

**Pull request merged**